### DISCUSSION TITLE
App Groups: Wildcards in program path didn't work

### DISCUSSION THREAD

**Discussion by Frankienul** [tnodir/fort#63] [Q&A] [answered]
it didn't work,
![image](https://user-images.githubusercontent.com/37241509/228399700-d68e4f74-054d-4499-9da5-78201ae98f58.png)

**Comment by tnodir** [comment, marked as answer]
Pattern format for wildcards is same as for [.gitignore](https://git-scm.com/docs/gitignore#_pattern_format).

The double asterisk `**` should be used with slash `/`:
> **/msedge.exe

**Reply by Frankienul** [reply]
it didn't work too.
![image](https://user-images.githubusercontent.com/37241509/228999267-6aab6bfb-6903-4cba-bedd-27634b84d7a5.png)

**Reply by tnodir** [reply]
Remove `msedge.exe` program from Programs window.

**Reply by Frankienul** [reply]
i don't know why.
![image](https://user-images.githubusercontent.com/37241509/229046570-9b598b0e-86ff-4cd1-86fd-0895d3630664.png)

**Reply by tnodir** [reply]
@Frankienul There is Programs window, which you can open from tray menu.
Find msedge.exe in the Programs and delete it.
So the AppGroups settings will have effect for msedge.exe

**Reply by Frankienul** [reply]
i closed edge, then delete msedge.exe from Programs, but when i open edge again, fort-firewall allow edge connect website, why?
![image](https://user-images.githubusercontent.com/37241509/229072609-90f8a13c-c72e-4cce-8ef6-2270168d8e23.png)

**Reply by tnodir** [reply]
1. Remove `...\msedge.exe` from Programs
2. Add `**\msedge.exe` to App Group and Save

I've double checked it - works fine.
But there's a bug/feature now - you've to save Options after removing it from Programs.
